### PR TITLE
Additional mac PCSC support

### PR DESCRIPTION
**Pull request by kuznetsovin**
Correct binding for Mac OS X framework for PCSC.

### CODE REVIEW

File: scard_darwin.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
+// +build darwin
+
+package scard
+
+// BUG(mg): Does not work on darwin. (older/different libpcsclite?)
```
Comment (ebfe) [inline scard_darwin.go]: That can be removed now?

**Comment by ebfe** [comment]
Thanks, this look good so far (though i don't have access to an osx machine to actually test anthing).  Seems there is a build problem on go 1.5 (https://travis-ci.org/ebfe/scard/jobs/118435396). If there is a easy workaround/fix for this it would be good to include.

**Comment by kuznetsovin** [comment]
I adding fix for type conflict go1.5. And create pull request.

**Comment by kuznetsovin** [comment]
Testing example working well with these fix.

File: scard_darwin.go (hunk 1 above)
Comment (kuznetsovin) [inline scard_darwin.go]: In new commit this removed.

**Comment by ebfe** [comment]
Thanks

**Pull request merged**